ground/src/main/java/com/google/android/ground/persistence/local/LocalValueStore.kt

```diff
@@ -132,6 +132,14 @@ class LocalValueStore @Inject constructor(private val preferences: SharedPrefere
     }
   }
 
+  fun setDataSharingTerms(surveyId: String, consent: Boolean) {
```

Comment:
Nit: Since this field holds whether a user granted content for a particular survey's terms, I think `setDataSharingConsent(surveyId: String, granted: Boolean)` would be more accurate here.

That would also help differentiate this from the actual terms themselves, which live in the survey.